code/style.css

```diff
@@ -1,7 +1,315 @@
 
-.active + .answer {
-  /* some styling here */
+body {
+  background: #060606;
+  color: #fff;
+  font-family: "Montserrat";
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+a:hover {
+  color: #FF33F6;
+}
+
+h1 {
+  font-size: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  text-align: center;
+  font-size: 40px;
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.italic {
+  font-style: italic;
+  font-size: 11px;
+}
+
+.uc {
+  text-transform: uppercase;
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+/* ------------------------- HEADER ------------------------- */
+header {
+  height: 15vh;
+}
+
+video {
+  object-fit: cover;
+  width: 100vw;
+  height: 80vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  margin-right: 30px;
+}
+
+header img {
+  width: 500px;
+}
+
+header nav a {
+  font-size: 12px;
+  letter-spacing: 10px;
+  margin-left: 30px;
+  padding-bottom: 5px;  
+}
+
+
+/* ------------------------- HERO ------------------------- */
+.hero {
+  text-align: center;
+  height: 65vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.hero p {
+  font-size: 18px;
+  max-width: 45vw;
+}
+
+.hero-arrow-down {
+  width: 50px;
+  height: 50px;
+  border: 1px solid #fff;
+  border-radius: 50%;
+  margin-top: 20px;
+}
+
+.hero-arrow-down:hover {
+  border: 1px solid #FF33F6;
+}
+
+.hero-arrow-down p {
+font-size: 30px;
+}
+
+
+/* ------------------------- MARQUEE ------------------------- */
+.marquee {
+  margin-top: 20px;
+}
+
+
+/* ------------------------- SECRETFORM ------------------------- */
+.secret-form {
+  width: 80vw;
+  margin: 50px auto;
+  text-align: center;
+  border: 1px solid #fff;
+  padding: 40px;
+}
+
+label {
+  display: block;
+  margin: 20px 0 5px 0;
+}
+
+textarea, input {
+  border: 0;
+  outline: none;
+  border: 1px solid #fff;
+  font-family: monospace;
+}
+
+textarea {
+  min-height: 150px;
+  width: 65vw;
+}
+
+input {
+  height: 30px;
+  width: 250px;
+}
+
+input:focus, textarea:focus {
+  border: 2px #FF33F6 solid;
+}
+
+.checkboxes-all-wrapper {
+  margin: 30px 0;
+}
+.checkbox-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.checkbox-wrapper label {
+  padding: 0;
+  margin: 0;
+}
+
+.checkbox {
+  height: auto;
+  width: auto;
+  margin-left: 10px;
+}
+
+button {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  display: block;
+  margin: 40px auto 20px auto;
+  border: 1px solid #fff;
+  color: #fff;
+  background: none;
+  outline: none;
```

Comment:
I know it's sooo tempting to remove the outline (they're not pretty), but don't. It makes your page less web accessible, when you tab through your site you can't see when the button is highlighted 😭 This is important for people using screen readers or cant use a mouse when browsing your page. I believe you will talk more about web accessibility in a few weeks 👍